io.home-assistant.heat/manifest.tt

```diff
@@ -0,0 +1,13 @@
+abstract class @io.home-assistant.heat
+#_[thingpedia_name="Heat Binary Sensor"]
+#_[thingpedia_description="Interface for Home Assistant's Heat Binary Sensor."]
+#_[description="Interface for Home Assistant's Heat Binary Sensor."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(hot,normal), out value : String)
+  #_[canonical="heat sensor state"]
+  #_[confirmation="the state of your heat sensor"]
+  #_[formatted=["The heat sensor is ${state}."]];
```

Comment:
To be honest, the fact that Home Assistant models heat sensors as "cold sensors" vs "hot sensors" (instead of a goldlocks 3-way) is puzzling, and makes me wonder whether we should merge them too.
Also, again "value"?